# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../devilspie/libwnck.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur devilspie-0.22-orig/src/xutils.c devilspie-0.22/src/xutils.c
--- devilspie-0.22-orig/src/xutils.c	2009-02-17 17:06:54.000000000 +0000
+++ devilspie-0.22/src/xutils.c	2009-02-17 17:07:59.000000000 +0000
@@ -305,9 +305,6 @@
   case WNCK_WINDOW_DIALOG:
     atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_DIALOG");
     break;
-  case WNCK_WINDOW_MODAL_DIALOG:
-    atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_MODAL_DIALOG");
-    break;
   case WNCK_WINDOW_TOOLBAR:
     atom = my_wnck_atom_get ("_NET_WM_WINDOW_TYPE_TOOLBAR");
     break;
